### PR TITLE
Test 2.2 on all laravel versions

### PR DESCRIPTION
**Pull request by eigan**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,35 +2,42 @@ name: CI
 on: [push, pull_request]
 jobs:
   tests:
-    name: Tests
-    runs-on: ${{ matrix.os }}
+    runs-on: 'ubuntu-latest'
+    name: PHP ${{ matrix.php }} - Laravel ${{ matrix.laravel }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
```
Comment (dpslwk) [inline .github/workflows/ci.yml:9]: why drop testing on macOS?

my dev is done on macOS though production is debian

File: composer.json [hunk 2]
```diff
@@ -17,15 +17,15 @@
...
+    "php": "^7",
     "doctrine/migrations": "~2.2",
-    "illuminate/config": "^8.0",
-    "illuminate/contracts": "^8.0",
-    "illuminate/console": "^8.0",
-    "laravel-doctrine/orm": "^1.7"
+    "illuminate/config": "^6|^7|^8" ,
+    "illuminate/contracts": "^6|^7|^8",
+    "illuminate/console": "^6|^7|^8",
+    "laravel-doctrine/orm": "^1"
```
Comment (dpslwk) [inline composer.json]: minor version spec should be include still (7.2.5 matches L6 & L7)
```
    "php": "^7.2.5",
    "doctrine/migrations": "~2.2",
    "illuminate/config": "^6.0|^7.0|^8.0" ,
    "illuminate/contracts": "^6.0|^7.0|^8.0",
    "illuminate/console": "^6.0|^7.0|^8.0",
    "laravel-doctrine/orm": "^1.5"
 ```
 
 and then for php 8 testing we just tag on `|8.0`
```
    "php": "^7.2.5|8.0",
    "doctrine/migrations": "~2.2",
    "illuminate/config": "^6.0|^7.0|^8.0" ,
    "illuminate/contracts": "^6.0|^7.0|^8.0",
    "illuminate/console": "^6.0|^7.0|^8.0",
    "laravel-doctrine/orm": "^1.5"
 ```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (eigan) [reply .github/workflows/ci.yml:9]: None of [doctrine/migrations](https://github.com/doctrine/migrations/blob/3.0.x/.github/workflows/continuous-integration.yml), [doctrine/orm](https://github.com/doctrine/orm/blob/2.8.x/.github/workflows/continuous-integration.yml), and [laravel/framework ](https://github.com/laravel/framework/blob/8.x/.github/workflows/tests.yml)tests with macos. Seems weird that we should support it.

Comment (dpslwk) [reply .github/workflows/ci.yml:9]: Fair enough

File: composer.json (hunk 2 above)
Comment (Jurigag) [reply composer.json]: Better tag ^8.0, you don't want to support only 8.0.

When you will add ^8.0? Can't test it at this moment.

Comment (eigan) [reply composer.json]: @Jurigag released now

**Pull request merged**